packages/currency/src/erc20/networks/fantom.ts

```diff
@@ -6,4 +6,54 @@ export const supportedFantomTokens: TokenMap = {
     symbol: 'YEL',
     decimals: 18,
   },
+  '0x6a07A792ab2965C72a5B8088d3a069A7aC3a993B': {
```

Comment:
I'm not sure it's necessary to add these tokens here? the currency API should be enough

Reply:
Oh man, I was having issues because I added them in both places and it was breaking because of duplicates.
I'll add them to the currency manager only then